lib/order.rb

```diff
@@ -0,0 +1,33 @@
+require './lib/menu.rb'
+
+class Order
+
+attr_reader :list_of_dishes, :selected_items
+
+def initialize(m = Menu.new)
+  @list_of_dishes = m
+  @selected_items = []
+end
+
+def view_dishes
+  @item = @list_of_dishes.list
+end
+
+def select_dishes(item, quantity)
+  @total = 0
+  i = 0
+  selection = @list_of_dishes.list.select do |hash|
+  hash[:name] == item
```

Comment:
anywhere we use a do end block, it's convention to indent the code which sits inside of it

So:

```ruby
my_array.each do |item|
   puts item
end
```